report/2.Analysis/ProblemStatement.tex

```diff
@@ -1,11 +1,11 @@
 \section{Problem statement}
 Based on the analysis the following problem statement was defined:
 \begin{center}
-	How can a programming language that eases development for the Arduino platform be developed?
+	How can a programming language be implemented, that introduces novices to programming on the Arduino architecture, while keeping the syntax english-like?
```

Comment:
Ingos problem med den gamle sætning var, at den er for uklar. "how can a language be implemented" er, selvom vi har den indskudte sætning bagefter, meget uklar.

Vi bør måske skrive:
"In short, the problem is that novices have trouble with blah blah blah.......
Will an english-like syntax improve this?"
eller noget i den retning (den eneste måde at svare på denne sætning er gennem user-tests. Hvis vi ikke gider det, skal vi selvfølgelig ikke bruge præcis denne formulering).

Lige meget hvad, bør sætningen omskrives, fordi den indskudte sætning kan skrives meget bedre.